darts/models/forecasting/lgbm.py

```diff
@@ -163,6 +175,53 @@ def fit(
             Additional kwargs passed to `lightgbm.LGBRegressor.fit()`
         """
 
+        if categorical_past_covariates:
```

Comment:
There is a bit of code duplication here, this could be avaoided by iterating through the three covariate types/names in a zipped fashion

Reply:
Gave this a go for past and future covs. For static covs the check is slightly different so I'm handling that one still separately. Is this what you had in mind?